tests/test_call_nested_rtsequence.py

```diff
@@ -0,0 +1,65 @@
+from niveristand import nivs_rt_sequence, NivsParam, realtimesequencetools
+from niveristand.clientapi import DoubleValue
+from niveristand.library import wait
+import pytest
+
+
+@ NivsParam('var1', DoubleValue(0), NivsParam.BY_REF)
+@ nivs_rt_sequence
+def nested_sequence(var1):
+    wait(var1)
+
+
+@NivsParam('param', DoubleValue(0), NivsParam.BY_REF)
+@nivs_rt_sequence
+def call_nested_sequence_after(param):
+    wait(param)
+    nested_sequence(param)
+
+
+@NivsParam('param', DoubleValue(0), NivsParam.BY_REF)
+@nivs_rt_sequence
+def call_nested_sequence_before(param):
+    nested_sequence(param)
+    wait(param)
+
+
+@NivsParam('param', DoubleValue(0), NivsParam.BY_REF)
+@nivs_rt_sequence
+def call_nested_sequence_twice(param):
+    wait(param)
+    nested_sequence(param)
+    nested_sequence(param)
+    wait(param)
+    wait(param)
+
+
+@NivsParam('param', DoubleValue(0), NivsParam.BY_REF)
+@nivs_rt_sequence
+def call_complex_nested_sequences(param):
+    wait(param)
+    call_nested_sequence_after(param)
+    wait(param)
+    call_nested_sequence_before(param)
+    wait(param)
+
+
+run_tests = [
+    call_nested_sequence_after,
+    call_nested_sequence_before,
+    call_nested_sequence_twice,
+    call_complex_nested_sequences
+]
+
+
+def idfunc(val):
+    try:
+        return val.__name__
+    except AttributeError:
+        return str(val)
+
+
+@pytest.mark.parametrize("func_name", run_tests, ids=idfunc)
+def test_save_py_as_rts(func_name):
+    realtimesequencetools.save_py_as_rtseq(func_name,
+            r"C:\Users\Public\Documents\National Instruments\NI VeriStand 2020\Stimulus Profiles\Python Examples")
```

Comment:
This will fail because that path doesn't exist on the CI machine. Use the directory creation from test_saveasrtseq.py. You can move that to a utils to make it available for other tests. Isn't it enough to transform the python code to rtseq to test the functionality?

Reply:
I changed the test to just transform the python code to rtseq, without saving it